llvm/test/CodeGen/SPIRV/debug-info/basic-global-di.ll

```diff
@@ -0,0 +1,129 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+
+source_filename = "example.c"
+target datalayout = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-G1"
+target triple = "spir"
```

Comment:
Are target triple and data layout needed to be defined explicitly in the source? Triple is also passed as an argument to llc above.

Reply:
Done.